db/data_migration/20150427095111_archive_all_worldwide_priorities.rb

```diff
@@ -0,0 +1,4 @@
+WorldwidePriority.where(state: "published").each do |wp|
+  puts "Archiving #{wp.title} - edition #{wp.latest_edition.id}"
+  wp.archive!
```

Comment:
So this won't actually mark the documents as archived on GOV.UK - there is no `Unpublishing` record and they won't have been removed from search either. You'll want to go through `Whitehall.edition_services.archiver`. This will make sure all the service listeners are triggered for unpublishing actions, e.g. removing from the search index, e.g. something like this:

```ruby
reason = "This shows the policy of the 2010 to 2015 Conservative and Liberal Democrat coalition government"
edition.build_unpublishing(explanation: reason, unpublishing_reason_id: UnpublishingReason::Archived.id)
archiver = Whitehall.edition_services.archiver(edition)
unless archiver.perform!
  puts "Could not archive edition: #{archiver.failure_reason}"
end
```